images/fcos-bastion-image/root/usr/share/bastion_services/dhcp/dnsmasq.conf

```diff
@@ -0,0 +1,704 @@
+interface=baremetal
+dhcp-range=192.168.90.100,192.168.90.149,6h
+enable-ra
+dhcp-range=fd99:2222:3456::100,fd99:2222:3456::200,6h
+
+dhcp-match=set:efi,option:client-arch,7
+dhcp-match=set:efi,option:client-arch,9
+dhcp-match=set:efi_arm,option:client-arch,11
+dhcp-boot=tag:efi,grubx64.efi
+dhcp-boot=tag:efi_arm,grubaa64.efi
+dhcp-option=tag:efi,18,x86_64
+dhcp-option=tag:efi_arm,18,aarch64
+enable-tftp
+tftp-root=/var/lib/tftpboot
+dhcp-option=6,192.168.90.1
+dhcp-option=option6:dns-server,[::]
```

Comment:
Is this ok or should it be `fd99:2222:3456::1`?

Is there a similar option for the ntp-server at line 20?

Reply:
Yes, added the ip for both dns and ntp server